pennylane/data/data_manager.py

```diff
@@ -15,27 +15,60 @@
 Contains the Dataset utility functions.
 """
 # pylint:disable=too-many-arguments,global-statement
-from concurrent.futures import ThreadPoolExecutor, wait
+from collections.abc import Iterable
+from concurrent.futures import ThreadPoolExecutor, wait, FIRST_EXCEPTION
 import os
+from urllib.parse import quote
 
 import requests
 from pennylane.data.dataset import Dataset
 
 S3_URL = "https://xanadu-quantum-datasets-test.s3.amazonaws.com"
 FOLDERMAP_URL = os.path.join(S3_URL, "foldermap.json")
-DATA_STRUCT_URL = os.path.join(S3_URL, "data_struct.json")
+DATA_STRUCT_URL = os.path.join(S3_URL, "data_struct_new.json")
```

Comment:
Why new?

Reply:
In short, the old `data_struct.json` is still in S3, but I was testing a new format for the file. I will replace it once this new design is approved.

thanks for catching this! The `_new` one only has three keys per data name: params, attributes, docstr. I've removed the types and the docstrings per-attribute, since we will always set the full docstring for the user. Now, data_struct[data_name]['docstr'] is just the entire docstring. No processing needed at the client anymore, we just set the `Dataset.__doc__` to be this value directly! Once approved,